engine-types/src/lib.rs

```diff
@@ -7,6 +7,7 @@ pub mod account_id;
 pub mod parameters;
 pub mod storage;
 pub mod types;
+pub mod types_new;
```

Comment:
Seems redundant? Why would we need this? Should just live in `types` ?

Reply:
After #386 merge, it will be renamed to `types`. In #386 it's a module, and `types.rs` removed.
To avoid merge conflicts I decided set some temporary name. It'll easily rename to `types` after #386 merge.

Reply:
For now, it was merged & deleted.